.vib/java/goss/java.yaml

```diff
@@ -5,7 +5,7 @@ command:
   # - In Java 1.8, the printed version is 1.8.0_x, but $APP_VERSION is formated as 1.8.x
   # - Versions are usually printed as x.y.z+b, but $APP_VERSION is formated as x.y.z-b
   check-app-version:
-    exec: java -version 2>&1 | sed "s/+/-/g" | sed "s/0_//g" | grep "$(echo $APP_VERSION | sed -E 's|([0-9]+)\.0\.0$|\1|g')"
+    exec: java -version 2>&1 | sed "s/+/-/g" | sed "s/0_//g" | grep "$(echo $APP_VERSION | sed -E 's|([0-9]+)\.0\.0$?|g')"
```

Comment:
Did you try this?
`\1`  is a backreference that refers to the first captured group.
```
    exec: java -version 2>&1 | sed "s/+/-/g" | sed "s/0_//g" | grep "$(echo $APP_VERSION | sed -E 's|([0-9]+)\.0\.0$?|\1|g')"
```

Reply:
Updated! Thanks!